### ISSUE TITLE
Compress / Decompress  Number Divergence

### ISSUE THREAD

**Issue by manu-de-verdun** [beenotung/compress-json#3]
Hi,

Just find one issue during the compress / decompress process, in node

perscore values are different after completing a cycle

(the input is the result of a BigQuery query)

a workaround was found using Round in the SQL query to limit the number size

Input: 
```
[
 {
  "sub": 1,
  "factors": [
   {
    "name": 0,
    "metric": 15,
    "percscore": 0.12371134020618557
   },
   {
    "name": 1,
    "metric": 4,
    "percscore": 0.032989690721649485
   },
   {
    "name": 2,
    "metric": 22,
    "percscore": 0.18144329896907216
   },
  ],
  "avg_score": 2.1651785714285716,
 }
]
```

outPut

```
[
 {
  "sub": 1,
  "factors": [
   {
    "name": 0,
    "metric": 15,
    "percscore": 0.03371134020618557
   },
   {
    "name": 1,
    "metric": 4,
    "percscore": 0.002989690721649485
   },
   {
    "name": 2,
    "metric": 22,
    "percscore": 0.4814432989690722
   },
  ],
  "avg_score": 2.1651785714285716,
 }
] 
```

thanks

**Comment by beenotung** [comment]
The reported bug is fixed https://github.com/beenotung/compress-json/commit/3f3bc2f41451a3573255bc30f54643a5af836545 and and patched version is release as compress-json@1.0.5

@manu-de-verdun you may close the issue if confirmed

**Comment by beenotung** [comment]
I'm closing this issue because I think it has been resolved.
You're welcomed to re-open the issue if you're still encountering related bugs.
Thank you.
